Scoop/Controller/Home/HomeTableViewCell.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  HomeTableViewCell.swift
+//  Scoop
+//
+//  Created by Reade Plunkett on 4/21/22.
+//
+
+import UIKit
+
+class HomeTableViewCell: UITableViewCell {
+    
+    private let containerView = UIView()
+    private let titleLabel = UILabel()
+    private let depatureLabel = ImageLabelView()
+    private let arrivalLabel = ImageLabelView()
+    private let dateLabel = ImageLabelView()
+    private let dotsImageView = UIImageView()
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        selectionStyle = .none
+        
+        setupContainerView()
+        setupDateLabel()
+        setupTitleLabel()
+        setupDepartureLabel()
+        setupDotsImageView()
+        setupArrivalLabel()
+    }
+    
+    private func setupContainerView() {
+        containerView.clipsToBounds = true
+        containerView.layer.masksToBounds = true
+        containerView.layer.cornerRadius = 10
+        containerView.layer.borderWidth = 2
+        containerView.layer.borderColor = UIColor.borderColor.cgColor
+        contentView.addSubview(containerView)
+        
+        containerView.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview()
+            make.top.bottom.equalToSuperview().inset(10)
+        }
+    }
+    
+    private func setupDateLabel() {
+        dateLabel.imageView.image = UIImage(systemName: "calendar")
+        dateLabel.label.font = .systemFont(ofSize: 16)
+        containerView.addSubview(dateLabel)
+        
+        dateLabel.snp.makeConstraints { make in
+            make.top.trailing.equalToSuperview().inset(10)
+        }
+    }
+    
+    private func setupTitleLabel() {
+        titleLabel.font = .systemFont(ofSize: 18, weight: .bold)
+        titleLabel.adjustsFontSizeToFitWidth = true
+        titleLabel.textColor = .black
+        containerView.addSubview(titleLabel)
+        
+        titleLabel.snp.makeConstraints { make in
+            make.leading.top.equalToSuperview().inset(10)
+            make.trailing.lessThanOrEqualTo(dateLabel.snp.leading).offset(-10)
```

Comment:
inset instead of offsetting by negative

Reply:
That wouldn't work

Reply:
it would set the trailing anchor to be at an 10 pixel inset of the dateLabel, so it would partially cover the date label
![simulator_screenshot_88117D1A-9DF5-4CF0-B897-248528E629DE](https://user-images.githubusercontent.com/19507508/165310396-6e69f7ec-48a9-43ca-8f90-884cdb8a9d63.png)
.